Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps-01-code-0005.swift

```diff
@@ -1,18 +1,58 @@
 import ComposableArchitecture
 import SwiftUI
 
-@main
```

Comment:
📓 Due to the insertion of a new "0004" code snippet above, this one was the old "0004" and is now diffed with the previous "0005" and so on with the following files… So it is quite difficult to review here… Sorry